tests/jobserver/test_admin.py

```diff
@@ -0,0 +1,28 @@
+import pytest
+from django.contrib.admin.sites import AdminSite
+from django.urls import reverse
+
+from jobserver.admin import UserAdmin
+from jobserver.models import User
+
+from ..factories import UserFactory
+
+
+@pytest.mark.django_db
+def test_useradmin_approve_users(rf, superuser):
+    UserFactory.create_batch(5)
+    users = User.objects.all()
+
+    user_admin = UserAdmin(model=User, admin_site=AdminSite())
+
+    request = rf.get("/")
+    request.user = superuser
+
+    response = user_admin.approve_users(request, users)
+
+    assert response.status_code == 302
+
+    base = reverse("approve-users")
+    query = [f"user={u.pk}" for u in users]
```

Comment:
Could this be flaky?  Is it worth explicitly ordering the users to make this deterministic?